tools/go-cli/go-whisk-cli/commands/namespace.go

```diff
@@ -78,7 +78,7 @@ var namespaceGetCmd = &cobra.Command{
         namespace, _, err := client.Namespaces.Get(qName.namespace)
         if err != nil {
             whisk.Debug(whisk.DbgError, "client.Namespaces.Get(%s) error: %s\n", qName.namespace, err)
-            errStr := fmt.Sprintf("Unable to obtain namespace entities for namespace '%s': %s", qName.namespace, err)
+            errStr := fmt.Sprintf("Unable to obtain namespace entities for namespace '%s'", qName.namespace)
```

Comment:
This just hides the fact that the API host was not set and is arguably more confusing to the end user. Instead the CLI should check that the API host is not set and report an error message along the lines of "Cannot perform operation because an API host is not set" instead of reporting this: "Request failure: Get https:///api/v1/namespaces/_/: http: no Host in request URL".